deps/http_parser.mk

```diff
@@ -0,0 +1,39 @@
+## http_parser
+
+HTTPPARSER_GIT_URL := git://github.com/nodejs/http-parser.git
+HTTPPARSER_TAR_URL = https://api.github.com/repos/nodejs/http-parser/tarball/$1
+$(eval $(call git-external,http_parser,HTTPPARSER,,,$(BUILDDIR)))
+
+# Manually set the PLATFORM flag to whatever $(OS) is, because at least one
+# person (@staticfloat) has the PLATFORM envvar set to something else.
+# Also manually set PREFIX to DESTDIR
+HTTPPARSER_FLAGS := PLATFORM=$(shell echo $(OS) | tr A-Z a-z) PREFIX=$(build_prefix)
+
+$(eval $(call staged-install, \
```

Comment:
[Yes](https://github.com/JuliaLang/julia/blob/548465b6ded7d2d4f908760e66329d7fcdfd356d/deps/tools/common.mk#L159-L163) and [Yes](https://github.com/JuliaLang/julia/blob/548465b6ded7d2d4f908760e66329d7fcdfd356d/deps/tools/common.mk#L132-L134)